### DISCUSSION TITLE
Addition of example `.ipynb` files to `.gitignore`.

### DISCUSSION THREAD

**Discussion by tplobo** [Fusion-Power-Plant-Framework/bluemira#744] [Ideas]
Is there any reason the `.ipynb` extension in the examples folder is not in the list of files ignored by GIT? In my understanding these files are only temporary and should not be added automatically to any branches in the remote, since the `convert_py_to_ipynb.py` script already creates them locally.

**Comment by je-cook** [comment]
the `.ipynb` files are autogenerated by the script but currently they are not automatically added to the commit. The user has to add the file themselves. 
My main reasoning is that sometimes the script is not generated properly (it is not a robust conversion) therefore a check of the output may be needed before commiting. 
Adding notebooks to the gitignore would then make it more likely people would forget to add the files in the first place

**Reply by tplobo** [reply]
But I don't mean all .ipynb files, just the notebooks in the examples folder.

**Reply by DanShort12** [reply]
It's for convenience really. We could just get the user to generate the ipynb files themselves, but better we generate them in a way that we know is right rather than the users trying to do it themselves when they'll likely be unfamiliar with the codebase. It also means that in the future we could have online interactive examples via e.g., [binder](https://mybinder.readthedocs.io/en/latest/) once we have an available docker image.

**Comment by je-cook** [comment]
Notebooks are now not stored on the repo since the addition and use of jupytext
